src/app/login/login.component.scss

```diff
@@ -0,0 +1,114 @@
+@import "../../main-styles.scss";
+
+.centered-content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  margin-top: 0px;
+  border-top-width: 200px;
+  border-top-style: solid;
+  border-bottom-width: 200px;
+  border-bottom-style: solid;
+}
+
+.home-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  max-width: 720px;
+  max-height: 200px;
+  padding-right: 15px;
+  padding-left: 15px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.home-selected-container {
+  display: flex;
+  flex-direction: column;
+  // flex-grow: 1;
+  padding: 40px;
+  border-radius: 8px;
+  width: 60%;
+  border-style: solid;
+}
+
+// Title
+.title {
+  font-family: "Lato";
+  font-size: 24px;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+}
+
+// Outlines the input boxes
+.input {
+    border-color: black;
+    border-width: 1px;    
+}
+
+// Each input field surrounded by div
+.control {
+
+
+  &:focus {
+    transform: scale(0.95);
+    background-color: #d9e8ff;
+  }
+
+  &:hover {
+    // box-shadow: 0 0 0 5px #eeeeee;
+    transform: scale(0.95);
+    transition: box-shadow 0.2s ease, transform 0.2s ease;
+  }
+}
+
+// Errors that show up when field is incomplete
+.help-block {
+    color:  #ff0000;
+}
+
+// Login Button 
+button {
+  margin-right: 200px;
+  border-radius: 3px;
+  margin-top: 10px;
+  margin-bottom: 5px;
+
+  &:hover {
```

Comment:
:) Such an easy CSS addition that makes it look really swanky!

Reply:
Yep, I'm a fan and sucker for this kind of stuff.